cves/CVE-2016-2327.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  An out-of-bounds array access could be triggered by a user crafting a custom .avi file that would cause an error
+  in performing row calculations.
```

Comment:
Would really like to know more about the final product. What happens when the index goes out of bounds? System downtime? Stack trace printed?